lib/internal/vm/module.js

```diff
@@ -224,6 +243,15 @@ class Module {
   }
 }
 
+function validateInteger(prop, propName) {
+  if (!Number.isInteger(prop)) {
+    throw new ERR_INVALID_ARG_TYPE(propName, 'integer', prop);
+  }
+  if ((prop >> 0) !== prop) {
+    throw new ERR_OUT_OF_RANGE(propName, '32-bit integer', prop);
+  }
+}
```

Comment:
Does this not belong somewhere a little more generic perhaps? Somewhat surprising there isn't already something for this too?

Reply:
I agree, especially since I copied this function from another file. I'll look into bringing together type validation functions in another PR if that's fine for you.